include/drivers/sensor.h

```diff
@@ -294,6 +301,17 @@ enum sensor_attribute {
 	 * algorithms to calibrate itself on a certain axis, or all of them.
 	 */
 	SENSOR_ATTR_CALIB_TARGET,
+	/** Configure the operating modes of a sensor. */
+	SENSOR_ATTR_CONFIGURATION,
+	/** Set a calibration value needed by a sensor. */
+	SENSOR_ATTR_CALIBRATION,
+	/**
+	 * Mask or Enable sensor functions usually related to
+	 * generating interrupts.
+	 */
+	SENSOR_ATTR_MASK,
+	/** Enable or Disable alerts and interrupts. */
+	SENSOR_ATTR_ALERT,
```

Comment:
If I didn't understand wrong, the `ALERT` attribute sets the alert threshold. It is enabled in the `MASK` register. Mask is also a very generic term. I suggest the following change to make these attributes potentially more useful for other sensors:
```suggestion
	/** Enable/disable sensor features */
	SENSOR_ATTR_FEATURE_MASK,
	/** Alert threshold or alert enable/disable */
	SENSOR_ATTR_ALERT,
```

Reply:
Done